theme/highcontrast.less

```diff
@@ -459,6 +473,14 @@
             }
         }
     }
+    .ui.card.link.newprojectcard{
+        background-color: #1B8800 !important;
```

Comment:
This will definitely change the color across the editors; this button is typically the same color as the download button.

https://github.com/microsoft/pxt/blob/095250200b6d106b1b5b3c0f81bd23c2693f1e9f/theme/themes/pxt/views/card.overrides#L151-L158

Maybe fine to darken it like the script manager card in hc mode instead of pegging to a different hue?

Reply:
good point good point I'll try that out!